src/messages.js

```diff
@@ -1,9 +1,11 @@
 export const MESSAGE = {
     browserDisconnected:            'The {userAgent} browser disconnected. This problem may appear when a browser hangs or is closed, or due to network issues.',
+    browsersDisconnected:           'The following browsers disconnected: {userAgents}. This problem may appear when a browser hangs or is closed, or due to network issues.',
```

Comment:
I feel like the fact that you can't run tests against disconnected browsers is a truism, though it fits well as a property name. So, I would go for something like:
`cantRunAgainstDisconnectedBrowsers: 'The following browsers disconnected: {userAgents}. Tests won't be run. This problem may appear when a browser hangs or is closed, or due to network issues.',`